MaloWBotLK/MaloWBotLK.lua

```diff
@@ -203,9 +221,9 @@ function mb_UpdateClassOrder()
 	local name = UnitName("player")
 	mb_classOrder = {}
 	table.insert(mb_classOrder, name)
-    local members = mb_GetNumPartyOrRaidMembers()
-    for i = 1, members do
-        local unit = mb_GetUnitFromPartyOrRaidIndex(i)
+	local members = mb_GetNumPartyOrRaidMembers()
+	for i = 1, members do
+		local unit = mb_GetUnitFromPartyOrRaidIndex(i)
```

Comment:
You have a lot of indentation changes like these that should not be part of this PR

Reply:
Reverted indentation changes.